USFMToolsSharp/Models/Markers/VAMarker.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace USFMToolsSharp.Models.Markers
+{
+    /// <summary>
+    /// Alternate verse number
+    /// </summary>
+    public class VAMarker : Marker
+    {
+        public string AltVerseNumber;
+        public string AltVerseCharacter
+        {
+            get
+            {
+                var firstCharacterMarker = GetChildMarkers<VPMarker>();
+                if (firstCharacterMarker.Count > 0)
+                {
+                    return firstCharacterMarker[0].VerseCharacter;
+                }
+                else
+                {
+                    return AltVerseNumber;
+                }
+            }
+        }
+        public override string Identifier => "va";
+        public override string PreProcess(string input)
+        {
+            AltVerseNumber = input;
+            return string.Empty;
+        }
+        public override List<Type> AllowedContents => new List<Type>()
+        {
+            typeof(VPMarker),

```

Comment:
vp should be in v not in va.

Reply:
At least that is what I read here. It should override what char the verse is using for it. So it should probably be wrapped up into the v marker's figuring out of what char it should display.

https://ubsicap.github.io/usfm/chapters_verses/index.html#vp-vp

Reply:
Since the \vp marker is used to establish their own number system... 

Would this be valid usfm? `\v 1 \vp a \vp* \va 3 \vp c \vp* \va* In the beginning.....`

Reply:
Actually yup that looks good. My bad.